plugin/app/feishu/feishu.go

```diff
@@ -502,28 +525,50 @@ func (p *Provider) GetIDByEmail(ctx context.Context, tokenCtx TokenCtx, emails [
 }
 
 func formatForm(content Content) (string, error) {
-	type form []struct {
+	type form struct {
 		ID    string `json:"id"`
 		Type  string `json:"type"`
 		Value string `json:"value"`
 	}
-	forms := form{
-		{
-			ID:    "1",
-			Type:  "input",
-			Value: content.Issue,
-		},
-		{
-			ID:    "2",
-			Type:  "input",
-			Value: content.Stage,
-		},
-		{
-			ID:    "3",
-			Type:  "textarea",
-			Value: content.Description,
-		},
+	var forms []form
+
+	forms = append(forms, form{
+		ID:    "1",
+		Type:  "input",
+		Value: content.Issue,
+	})
+
+	forms = append(forms, form{
+		ID:    "2",
+		Type:  "input",
+		Value: content.Link,
+	})
+
+	forms = append(forms, form{
+		ID:    "3",
+		Type:  "input",
+		Value: content.Stage,
+	})
+
+	var taskListValue strings.Builder
+
+	_, _ = taskListValue.WriteString(fmt.Sprintf("Stage %q has %d task(s).\n", content.Stage, len(content.TaskList)))
```

Comment:
Don't ignore error.

Reply:
WriteString always return nil error

Reply:
Let's still be correct. It's less pretty to see lots of error handling, but it's also confident to see errors are handled.